### DISCUSSION TITLE
[IntelliSense] : auto suggestion classnames when typing on object

### DISCUSSION THREAD

**Discussion by phatnguyen2009** [tailwindlabs/tailwindcss#14197] [Ideas]
i have object like this : 

`const myTestObject = {
  classIcon: 'bg-red-500  ', // Try typing here to see if suggestions appear
};`

and I setting tailwind config in setting.json in vscode : 

` "tailwindCSS.experimental.classRegex": [
    ["classIcon\\s*:\\s*[\"'`]([^\"'`]*)", "g"]
],`

but it not working ?

**Comment by wongjn** [comment]
Try

```json
"tailwindCSS.experimental.classRegex": [
  "classIcon\\s*:\\s*[\"']([^"']*)"
],
```
